featuretools/tests/entityset_tests/test_dask_es.py

```diff
@@ -2,66 +2,71 @@
 import pandas as pd
 import pytest
 
+import woodwork.logical_types as ltypes
+
 import featuretools as ft
 from featuretools.entityset import EntitySet
+from featuretools.tests.testing_utils import get_df_tags
 
 
-def test_create_entity_from_dask_df(pd_es):
+def test_add_dataframe(pd_es):
     dask_es = EntitySet(id="dask_es")
-    log_dask = dd.from_pandas(pd_es["log"].df, npartitions=2)
-    dask_es = dask_es.entity_from_dataframe(
-        entity_id="log_dask",
+    log_dask = dd.from_pandas(pd_es["log"], npartitions=2)
+    dask_es = dask_es.add_dataframe(
+        dataframe_id="log_dask",
         dataframe=log_dask,
         index="id",
         time_index="datetime",
-        variable_types=pd_es["log"].variable_types
+        logical_types=pd_es["log"].ww.logical_types,
+        semantic_tags=get_df_tags(pd_es["log"])
     )
-    pd.testing.assert_frame_equal(pd_es["log"].df, dask_es["log_dask"].df.compute(), check_like=True)
+    pd.testing.assert_frame_equal(pd_es["log"], dask_es["log_dask"].compute(), check_like=True)
 
 
-def test_create_entity_with_non_numeric_index(pd_es, dask_es):
+def test_add_dataframe_with_non_numeric_index(pd_es, dask_es):
     df = pd.DataFrame({"id": ["A_1", "A_2", "C", "D"],
                        "values": [1, 12, -34, 27]})
     dask_df = dd.from_pandas(df, npartitions=2)
 
-    pd_es.entity_from_dataframe(
-        entity_id="new_entity",
+    pd_es.add_dataframe(
+        dataframe_id="new_entity",
         dataframe=df,
         index="id")
 
-    dask_es.entity_from_dataframe(
-        entity_id="new_entity",
+    dask_es.add_dataframe(
+        dataframe_id="new_entity",
         dataframe=dask_df,
         index="id",
-        variable_types={"id": ft.variable_types.Id, "values": ft.variable_types.Numeric})
+        logical_types={"id": ltypes.Categorical, "values": ltypes.Integer},
+        semantic_tags={'id': 'foreign_key'})
```

Comment:
trying to keep the Id Variable type --> `'foreign_key'` semantic tag connection here, but I don't think this gets used for anything.

Reply:
In this particular case I think the `id` is the index, so I don't think we would want the `foreign_key` tag, no? Either wya, I'm thinking we don't need to worry about here though since it isn't relevant for the test.

Reply:
yeah, that makes sense. Happy to remove